Home.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>HOME</title>
+    </head>
+    <body style="background-color:powderblue;">
+        <style>
+            table, th, td {
+                border:1px solid black;
+            }
+        </style>
+
+        <h1 style="color:LightSeaGreen; font-size:300%;text-align:center;">HomePage</h1>
+        <p style="font-size:150%;text-align:center;">Usuários:</p>
+        <table style="width:100%">
+            <tr>
+                <th>ID</th>
+                <th>Nome</th>
+                <th>Telefone</th>
+                <th>Email</th>
+                <th>Data de Criação</th>
+            </tr>
+            <tr>
+                <td>0</td>
+                <td>0</td>
+                <td>0</td>
+                <td>0</td>
+                <td>0</td>
+            </tr>
```

Comment:
Could you fill these rows with data ? 
```suggestion
            <tr>
                <td>1</td>
                <td>John</td>
                <td>(11)98764-4321</td>
                <td>example@email.com</td>
                <td>20/10/2005</td>
            </tr>
```

Reply:
Done